_pages/cv.md

```diff
@@ -4,7 +4,7 @@ permalink: /cv/
 title: cv
 nav: true
 nav_order: 5
-cv_pdf: example_pdf.pdf
+cv_pdf: example_pdf.pdf # you can replace it with a link to your resume as well
```

Comment:
I think it should be more explicit, like `you can also use external links here`.